consultation_analyser/support_console/ingest.py

```diff
@@ -563,6 +565,71 @@ def import_questions(
         raise
 
 
+def import_cross_cutting_themes(consultation: Consultation, consultation_code: str, timestamp: str):
+    """
+    Import cross-cutting themes that span across multiple questions.
+    Must run after all themes have been imported.
+    """
+    logger.info(f"Starting cross-cutting themes import for consultation {consultation.title}")
+    
+    s3_client = boto3.client("s3")
+    cct_file_key = f"app_data/consultations/{consultation_code}/outputs/mapping/{timestamp}/cross_cutting_themes.json"
+    
+    try:
+        # Check if file exists
+        response = s3_client.get_object(Bucket=settings.AWS_BUCKET_NAME, Key=cct_file_key)
+        cct_data = json.loads(response["Body"].read())
+    except s3_client.exceptions.NoSuchKey:
+        logger.info("No cross_cutting_themes.json found, skipping cross-cutting themes import")
+        return
+    except Exception as e:
+        logger.error(f"Error reading cross-cutting themes file: {str(e)}")
+        raise
+    
+    # Get all questions for this consultation for lookup
+    questions_dict = {q.number: q for q in Question.objects.filter(consultation=consultation)}
+    
+    for cct_entry in cct_data:
+        # Create the cross-cutting theme
+        cross_cutting_theme = CrossCuttingTheme.objects.create(
+            consultation=consultation,
+            name=cct_entry["name"],
+            description=cct_entry["description"]
+        )
+        logger.info(f"Created cross-cutting theme: {cross_cutting_theme.name}")
+        
+        # Create assignments for each theme
+        assignments_to_create = []
+        for theme_ref in cct_entry["themes"]:
+            question_number = theme_ref["question_number"]
+            theme_key = theme_ref["theme_key"]
+            
+            # Find the question
+            question = questions_dict.get(question_number)
```

Comment:
```suggestion
            question = Question.objects.get(consultation=consultation, question_number)
```

There are very few Themes, Cross-Cutting-Themes so no need to fetch them all in advance?

Reply:
Current approach seems to be more consistent with loading all related objects into dictionaries and should result in fewer database queries

Reply:
fair